src/AppKit/NSAccessibility.cs

```diff
@@ -218,6 +218,7 @@ public static NSObject[] GetUnignoredChildren (NSObject originalChild)
 			return NSArray.ArrayFromHandle<NSObject> (handle);
 		}
 
+		[Mac (10, 9)]
```

Comment:
Maybe you want the new attribute instead, that one is deprecated IIRC

`[Introduced (PlatformName.MacOSX, 10, 9)]`